src/styles/page/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background: url(../images/star.svg) no-repeat center;
+    margin-right: 4px;
+  }
+
+  &--3 .stars__star:nth-child(-n + 3) {
+    background-image: url(../images/star-active.svg);
+  }
```

Comment:
What if it's a `stars--4` modifier?)
Now I don't see any stars in this case
<img width="203" alt="image" src="https://github.com/user-attachments/assets/c84a0865-37ce-4e7c-9c00-190411a8b6c0">